lms/djangoapps/grades/models.py

```diff
@@ -97,27 +98,14 @@ def from_json(cls, blockrecord_json):
             )
             for block in block_dicts
         )
-        return cls(record_generator)
+        return cls(record_generator, course_key)
 
     @classmethod
-    def from_list(cls, blocks):
+    def from_list(cls, blocks, course_key):
         """
         Return a BlockRecordList from a list.
         """
```

Comment:
yup.  already changed.  I'm waiting to push until Jenkins finishes it's round of python tests.